README.md

```diff
@@ -1,3 +1,4 @@
+
 # vost
 
 Aplicação para VOST
```

Comment:
This should probably be in english as well or have the whole README in both languages.